eos-update-server/docs/eos-update-server.conf.5

```diff
@@ -41,6 +49,21 @@ Whether to advertise and distribute updates over the local network. This may be
 \fItrue\fP or \fIfalse\fP. If \fItrue\fP, \fBeos\-update\-server\fP(8) and
 \fBeos\-updater\-avahi\fP(8) are enabled; otherwise, they will both refuse to
 advertise or distribute updates.
+\"
+.SH [Repository 0–9] SECTION OPTIONS
```

Comment:
Did you consider different naming of the sections? I'd like to propose the scheme similar to what git uses for the remotes:
```
[Repository "/ostree/repo"]
RemoteName="eos"

[Repository "/some/other/path"]
RemoteName="foo"
```

The upside of this scheme is that removing some repository section from the configuration file won't involve updating all the other repository sections (like it would do in the currently proposed scheme - write 10 repository sections and then remove `[Repository 0]`.

Later:
From the code I can see that the ordering is not really enforced. Basically index is used for setting the root path to `/<index>` for non-zero indexes and to an empty string for the zero index for backward compatibility. So instead we could have:
```
[Repository "main"]
Path=/ostree/repo
RemoteName=eos

[Repository "foo"]
Path=/some/other/path
RemoteName="foo"
```

so for the "main" repo server would set an empty root path and for other it would be `/<name>`. But it's probably bikeshedding at this point.